tracer/src/Datadog.Trace/Ci/Coverage/Models/Global/CoverageInfo.cs

```diff
@@ -87,4 +95,19 @@ protected void ClearData()
             }
         }
     }
+
+    public double GetTotalPercentage()
+    {
+        var coveragePercentage = Data[0];
+        if (double.IsNaN(coveragePercentage) || double.IsNegativeInfinity(coveragePercentage))
+        {
+            coveragePercentage = 0;
+        }
+        else if (double.IsPositiveInfinity(coveragePercentage))
+        {
+            coveragePercentage = 100;
+        }
```

Comment:
This logic is copy-pasted 4 times - prob makes sense to just extract a small helper, something like

```csharp
public static double EnsureValidPercentage(double value)
{
    if (double.IsNaN(value) || double.IsNegativeInfinity(value))
    {
        return 0;
    }
    else if (double.IsPositiveInfinity(value))
    {
        return 100;
    }

    return value;
}
```

Reply:
true, done in [710e28d](https://github.com/DataDog/dd-trace-dotnet/pull/4661/commits/710e28dfb58981be2695c65bb17bb884400410cc)